Dominos_Cast.toc

```diff
@@ -4,8 +4,8 @@
 ## Notes: A movable casting bar, with timer
 ## Author: Tuller, Goranaws
 ## Dependencies: Dominos
-## Version: 2
+## Version: 3
 mediaPanel.lua
-embeds.xml
```

Comment:
Embeds should be kept around.  In some cases, users can remove any embedded libraries that are used standalone.